front/src/modules/trainschedule/components/ManageTrainSchedule/ManageTrainScheduleMap/RenderPopup.tsx

```diff
@@ -17,6 +19,8 @@ type FeatureInfoClickType = {
 
 function RenderPopup() {
   const { t } = useTranslation(['operationalStudies/manageTrainSchedule']);
+  const osrdConfActions = useOsrdActions();
```

Comment:
I think it's better to pass only what we need to the `setPointIti` function instead of the entire conf
```suggestion
  const { updateOrigin, updateDestination, addVias, updateFeatureInfoClick } = useOsrdActions() as
    | operationalStudiesConfSliceActionsType
    | stdcmConfSliceActionsType;
```

Reply:
Done